Makefile

```
@@ -64,6 +64,8 @@ governance-validate-json-check:
 	python3 tools/validate_governance_reports.py --json > /tmp/governance_validation.json
 	python3 -c 'import json; p=json.load(open("/tmp/governance_validation.json", "r", encoding="utf-8")); assert p.get("status")=="passed", f"Validator JSON status not passed: {p}"; print("Validator JSON status is passed.")'
 
+governance-check: governance-test governance-validate governance-validate-json-check
+.PHONY: governance-setup governance-deps-check governance-lint governance-schema-validate governance-artifact-inventory governance-policy-test governance-validator-test governance-evidence-manifest governance-evidence-verify governance-evidence-schema governance-report governance-report-schema governance-check-generated
 governance-check: governance-test governance-reports-validate governance-validate-json-check
```

checkmake (0.3.2)

[warning] Target "governance-check" defined multiple times (lines 67 and 69).

(uniquetargets)